ui/client/documents/upload/DropArea.js

```diff
@@ -62,15 +61,77 @@ export const FileDropSelector = withStyles((theme => ({
     margin: 1,
     flexDirection: 'row'
   }
-})))(({classes, onFileSelect, onDropFilesRejected, acceptExtensions, mini, CTA, multiple=true}) => {
+})))(({
+  classes,
+  onFileSelect,
+  acceptExtensions,
+  mini,
+  CTA,
+  multiple = true,
+  disableSelector,
+}) => {
+  const [alertVisible, setAlertVisible] = useState(false);
+  const [alertMessage, setAlertMessage] = useState('');
+
+  const onDropFilesRejected = (fileRejections) => {
+    if (fileRejections.length > 10) {
+      setAlertMessage('The upload limit is 10 files. Please try again with fewer files.');
+      setAlertVisible(true);
+    } else {
+      setAlertMessage(fileRejections[0].errors[0].message);
+      setAlertVisible(true);
+    }
+  };
 
-  const {getRootProps, getInputProps, isDragActive} = useDropzone({
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({
```

Comment:
Second observation (mentioned earlier on a comment)- `useDropzone` hook  Seems to accept a `disabled` property. Is it possible to use that in this instance?

https://react-dropzone.js.org/#src

Reply:
There is also a `maxFiles`. Did you try this?

Reply:
I used disabled and maxfiles (both of which are a few lines below this comment), but I didn't see the noDrag property. I'll give that a try and see if it disables the drag and drop functionality. Without the eventlistener, even when disabled, if you drag a file onto the input it will then open the pdf as the page itself. Definitely not the behavior we want. I'll check out noDrag.

Reply:
On further exploration, it looks like `disabled` is indeed disabling the react-dropzone drag and drop behavior already, so noClick/noDrag aren't necessary. However, the eventListeners are necessary to prevent the default browser drag and drop behavior (which works anywhere on the site, but why would someone be dragging a PDF onto the landing page, I guess?), because we still have the input area that _looks_ like it should accept drag and drop.

Reply:
Oooh I see what you mean. Users will run into this awkward situation of dropping the file on the page (not the drop zone) and navigating away from Dojo into, possibly, a PDF preview page! Good catch!